iceoryx_hoofs/test/moduletests/test_objectpool.cpp

```diff
@@ -93,6 +91,22 @@ using FooPool = iox::cxx::ObjectPool<Foo, CAPACITY>;
 using Index_t = FooPool::Index_t;
 constexpr int NO_INDEX = FooPool::NO_INDEX;
 
+struct FooPoolWithPrivateMembersAccess : FooPool
+{
+    char* getFirstPtrWrap()
+    {
+        return getFirstPtr();
+    }
+    char* getLastPtrWrap()
+    {
+        return getLastPtr();
+    }
+
+    Index_t nextFreeWrap()
+    {
+        return nextFree();
+    }
```

Comment:
A `using FooPool::getFirstPtrWrap;` is shorter and does the same :)

Reply:
nice, done